client/layout/masterbar/logged-in.jsx

```diff
@@ -155,6 +156,10 @@ class MasterbarLoggedIn extends Component {
 		preload( 'me' );
 	};
 
+	goToCheckout = ( siteId ) => {
```

Comment:
In an iframe (which is how this will be used in wp-admin) we will need more than a URL to change the contents of the main page, which is why this became a callback.

Reply:
Makes sense to me then 👍